src/cluster_legacy.c

```diff
@@ -3319,7 +3328,7 @@ int clusterProcessPacket(clusterLink *link) {
         sender->flags |= CLUSTER_NODE_EXTENSIONS_SUPPORTED;
     }
 
-    /* Checks if the node supports light message hdr */
+    /* Checks if the node supports some flags. */
     if (sender) {
         if (flags & CLUSTER_NODE_LIGHT_HDR_PUBLISH_SUPPORTED) {
```

Comment:
"node supports some flags" is not correct.

Some flags indicate that the node supports light messages. Other flags indicate other things.

The CLUSTER_NODE_MY_PRIMARY_FAIL flag doesn't indicate support for a feature. If it's set, it means that the sender supports this feature and it's primary is failing now. If it's not set, we don't know if the sender supports this feature or not... So "support" is not the right word for this new flag.

```suggestion
    /* Store some flags about the sender. */
    if (sender) {
        /* Check if the node supports light message hdr */
        if (flags & CLUSTER_NODE_LIGHT_HDR_PUBLISH_SUPPORTED) {
```